draft-illyes-cbcp.md

```diff
@@ -89,7 +101,24 @@ standard signals defined by
 [RFC9110](https://www.rfc-editor.org/rfc/rfc9110#name-server-error-5xx), preferably also
 additional heuristics such as relative response time of the server.
 
+As such, crawlers should log already visited URL and how many requests they sent to a resource 
+as well as the respective HTTP status codes in the responses,
+especially if error occur, to avoid repeatedly crawling the same source.
+
+Generally, crawlers should avoid multiple concurrent requests to the same ressources
+and use a crawling rate that is no higher than 1 request per second or as specified in
+the REP protocol. Further, ressources should not be re-crawled more often than every 24 hours.
```

Comment:
This is not going to work. News sites' homepage is the hub that's hit about every 5 seconds because they care about having their fresh content in search and they publish often enough that having that 5s hit is critical.

In general any hard limit will not work I reckon, but happy to be proven wrong.

Reply:
Is this still useful to say?
```suggestion
the REP protocol. Further, ressources should not be re-crawled too frequently.
```